doc/accessibility/index.md

```diff
@@ -0,0 +1,118 @@
+
+### Web Content Accessibility Guidelines (WCAG)
+
+The Web Content Accessibility Guidelines (WCAG) are a set of guidelines for making web content accessible to people with disabilities. 
+
+There are various versions:
+
+* Version 2.2. is the current version as of December 2024.
+
+* Version 2.1. is the previous version which is more widely used because it's been around longer.
+
+There are various levels:
+
+* Level A is the minimum level of compliance
+ 
+* Level AA is the moderate level of compliance.
+  
+* Level AAA is the maximum level of compliance.
+
+TODO: we need to learn more about this area.
+
+### Accessible Rich Internet Applications (ARIA)
+
+The use of ARIA roles, states, and properties will be employed to improve accessibility for users with disabilities, particularly for dynamic content and advanced user interface controls (e.g., dropdowns, modal dialogs). These will be used in conjunction with semantic HTML elements where possible.
+
+TODO: we need to learn more about this area.
+
+### Multiple modality practices
+
+The use of multiple kinds of interaction modalities, such as a using a mouse pointer, or keyboard commands, or voice recognition. 
+
+TODO: we need to learn more about this area.
+
+### Accessibility software engineering
+
+We are evaluating accessibility software engineering practices and implementations, such as UI/UX design principles, and programming component frameworks and libraries.
+
+TODO: we need to learn more about this area.
+
+### Accessibility testing
+
+Accessibility testing will be a part of the development lifecycle. 
+
+We intend to leverage test automation tools, and accessibility compliance testing services, and if necessary manual testing, to identify and resolve accessibility issues before the release.
+
+We intend to do real-world user acceptance testing with individuals with disabilities.
+
+We intend to do continuous automatic testing to ensure that accessibility standards evolve with product iterations.
+
+TODO: we need to learn more about this area.
+
+### Training
+
+Developers, designers, and content creators will undergo accessibility training to ensure awareness of the standards and best practices. 
+
+This includes understanding how to create accessible content, design accessible interfaces, and properly implement ARIA.
+
+TODO: we need to learn more about this area.
+
+## Recommendations
+
+TODO
+
+## Consequences
+
+The system will be more inclusive and usable for a wider range of people, including those with disabilities, such as visual impairments, motor impairments, and cognitive impairments.
+
+- Increased legal and regulatory compliance, minimizing the risk of lawsuits or fines related to accessibility issues.
+
+- Potential additional development and maintenance costs due to the complexity of implementing and testing accessibility features.
+
+- Possible slower release cycles due to the time required for accessibility testing and refinements.
```

Comment:
Maybe make this focused on the improved quality resulting from the rigor, we're taking more time as we're building to a higher standard to the benefit of all users

Reply:
Good. Done.